src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -314,6 +314,10 @@ public abstract class Controller
     @Deprecated
     static final String STATIC_SCALING_FACTORS_OPTION = "static_scaling_factors";
 
+    static final String SHARD_MAX_SSTABLES_FACTOR_OPTION = "shard_max_sstables_factor";
```

Comment:
The name isn't very readable to me.

Maybe `max_sstables_per_shard_factor` or `sstables_per_shard_max_factor`?

Or `non_overlapping_threshold/trigger`?

Note that we also need to document this in `UnifiedCompactionStrategy.md` -- both as a section that describes the behaviour as well as a paragraph in the configuration section. If we can't find a default that works well in almost all situations, also make sure it is eventually added to the product documentation.

Reply:
done.